packages/btc/src/fungible.ts

```diff
@@ -0,0 +1,38 @@
+import { Config } from '@buildwithsygma/core';
+import type { BaseTransferParams, BitcoinTransferRequest } from 'types';
+
+import { BaseTransfer } from './base-transfer.js';
+
+export async function createBitcoinFungibleTransfer(
+  params: BaseTransferParams,
+): Promise<BitcoinTransfer> {
+  const config = new Config();
+  await config.init(process.env.SYGMA_ENV);
+  const transfer = new BitcoinTransfer(params, config);
+
+  return transfer;
+}
+
+class BitcoinTransfer extends BaseTransfer {
+  protected destinationAddress: string;
+  protected amount: bigint;
+
+  constructor(transfer: BaseTransferParams, config: Config) {
+    super(transfer, config);
+    this.destinationAddress = transfer.destinationAddress;
+    this.amount = transfer.amount;
+  }
+
+  getUriEncodedUtxoRequest(): string {
+    const { address } = this.resource;
+    return `bitcoin:${address}?amount=${this.amount}&destinationAddress=${this.destinationAddress}`;
```

Comment:
also tx opcodes should be encoded in message field

Reply:
> This isn't https://en.bitcoin.it/wiki/BIP_0021 ? Because there isn't destinationAddress field. Should this be `message?`

I was thinking that you can use

```
otherparam     = qchar *qchar [ "=" *qchar ]
```

to encode this `destinationAddress` if I understood correctly `otherparam` as your own defined param. Message could be more correct.

Reply:
> also tx opcodes should be encoded in message field

are you referring to the encoded data to pass in the `op_return` of signed transaction?

Reply:
>to encode this destinationAddress if I understood correctly otherparam as your own defined param. Message could be more correct.

but wallets won't recognize or use custom params. You need to use ones that are widely recognized and supported by wallets. That's why I'm saying you should use existing wallet in example to make sure it's going to work.

Reply:
Method was removed